src/keepers/HarvestResolverOptimism.sol

```diff
@@ -253,29 +254,21 @@ contract HarvestResolverOptimism is IResolver, Ownable {
             minimumAmountOut = minimumAmountOut - (minimumAmountOut * h.slippageBps) / SLIPPAGE_PRECISION;
 
             // Find out the expected exchange of OP to debt token
-            uint256[] memory expectedExchange = new uint256[](1);
+            uint256 expectedExchange;
             address[] memory token = new address[](1);
             token[0] = h.aaveToken;
             uint256 claimable = IRewardsController(0x929EC64c34a17401F460460D4B9390518E5B473e).getUserRewards(token, yieldToken, IRewardCollector(h.rewardCollector).rewardToken());
             // Find expected amount out before calling harvest
             if (IRewardCollector(h.rewardCollector).debtToken() == 0xCB8FA9a76b8e203D8C3797bF438d8FB81Ea3326A) {
-              // OP -> USDC -> alUSD
-              IVelodromeSwapRouter.route[] memory routes = new IVelodromeSwapRouter.route[](2);
-              routes[0] = IVelodromeSwapRouter.route(0x4200000000000000000000000000000000000042, 0x7F5c764cBc14f9669B88837ca1490cCa17c31607, false);
-              routes[1] = IVelodromeSwapRouter.route(0x7F5c764cBc14f9669B88837ca1490cCa17c31607, 0xCB8FA9a76b8e203D8C3797bF438d8FB81Ea3326A, true);
-              expectedExchange = IVelodromeSwapRouter(IRewardCollector(h.rewardCollector).swapRouter()).getAmountsOut(claimable, routes);
+              expectedExchange = claimable * uint(IChainlinkOracle(0x0D276FC14719f9292D5C1eA2198673d1f4269246).latestAnswer()) / 1e10;
             } else if (IRewardCollector(h.rewardCollector).debtToken() == 0x3E29D3A9316dAB217754d13b28646B76607c5f04) {
```

Comment:
why divide by 1e10?

Reply:
I did the math on paper and that was what it took to get the correct value. I will double check again

Reply:
Not my best attempt at math. Fixed now